webview/src/shared/components/iconMenu/IconMenu.tsx

```diff
@@ -46,18 +48,18 @@ export const IconMenu: React.FC<IconMenuProps> = ({ items }) => {
           setTooltipDisabled(false)
         }}
       >
-        <ul className={styles.menu} role="menu">
-          {items
-            .filter(({ hidden }) => !hidden)
-            .map(item => (
+        {visibleItems.length > 0 ? (
```

Comment:
Change upsetting the tests is in here

Reply:
Changing to an empty JSX element seems to do the trick:

```
        {visibleItems.length > 0 ? (
          <ul className={styles.menu} role="menu">
            {visibleItems.map(item => (
              <IconMenuItem
                {...item}
                key={item.tooltip}
                tooltipTarget={tooltipTarget}
                menuTarget={menuTarget}
              />
            ))}
          </ul>
        ) : (
          <></>
        )}
```

Reply:
thanks, trying it out

Reply:
done! let's see if that resolves it